test/integration/discordactions.test.js

```diff
@@ -464,7 +479,7 @@ describe("Discord actions", function () {
           expect(res.body.message).to.equal("An internal server error occurred");
           return done();
         });
-    });
+    }).timeout(10000);
```

Comment:
Why are we waiting for 10s here to complete the test?

Reply:
This extends timeout for the test. The test doesn't necessarily have to take 10s.

This has been done in a lot of other tests for this method to avoid the hard coded delay. Will solve this in a separate PR.

Reply:
Please create an issues an add the link here,
Approving it for now

Reply:
https://github.com/Real-Dev-Squad/website-backend/pull/1772